README.md

```diff
@@ -59,7 +59,7 @@ Reference
 
 ##### Credential Setup
 
-- [Read Only CFT](./cost/aws/FlexeraReadOnlyPolicy.template)
+- [Flexera Automation CloudFormation Template](./tools/cloudformation-template)
```

Comment:
@bryankaraffa - quick question, previously you were linking to the actual template, but now you are not (which makes sense as it now directs to the README). Just wanted to make sure this was intentional

Reply:
Yeah I think linking to README will be a better UX for customers.  README has links to the CFT, and also steps for how to use it (including 1-click deploy link)

Reply:
bump @nia-vf1 , let me know if there's any other Q's.  Thank you!